### PR TITLE
Refac/Remove assets

### LINKED ISSUE #5420: Caseflow styles, cleanup/refactor
As a prerequisite for #5266 

- Remove all unused styles
- Organize selectors and files 
- Potentially move some logic to component (button)
- deprecate caseflow/vendor/ move files to /assets
- add front-end color constants.
-tbd

### PR DESCRIPTION
**Pull request by nolawi**
Connects #5420

### Description
- [x] Removes `clipboard` jquery plugin
- [x] Removes erb certification header
- [x] Removes vendor/stylesheets, moves file
- [x] Removes/combines unessary selectors

### Acceptance Criteria 
- [ ] No changes to UI, and functionality

### Testing Plan

1. Certification Header (copy appeal id)
2. Test dropdown component `/test/users`
3. Reader/PDF viewer
4. test stats
5. test `//dispatch/work-assignments` table

### CODE REVIEW

File: app/assets/stylesheets/_rc_collapse.scss [hunk 1]
```diff
@@ -1,36 +1,38 @@
-/*
-* rc-collapse
-* Repo: https://github.com/react-component/collapse
-* This css has been modified to match the accordion for caseflow.
-*/
+//rc-collapse
+// Repo: https://github.com/react-component/collapse
+// //This css has been modified to match the accordion for caseflow.
```
Comment (nolawi) [inline app/assets/stylesheets/_rc_collapse.scss]: no changes in this file, just moved out of vendor

File: app/views/layouts/certifications.html.erb [hunk 2]
```diff
@@ -1,37 +0,0 @@
-<% content_for :content do %>
```
Comment (nolawi) [inline app/views/layouts/certifications.html.erb]: we now use the header component in cert

File: app/assets/stylesheets/_certification_v2.scss [hunk 3]
```diff
@@ -1,20 +0,0 @@
-@import 'uswds/lib/addons/font-stacks';
-@import 'uswds/core/variables';
-
-.form9-viewer {
-  height: 500px;
-}
```
Comment (nolawi) [inline app/assets/stylesheets/_certification_v2.scss]: added to component

File: app/assets/stylesheets/reader/_pdf_viewer.scss [hunk 4]
```diff
@@ -0,0 +1,355 @@
+
```
Comment (nolawi) [inline app/assets/stylesheets/reader/_pdf_viewer.scss]: no changes here, just converting to scss for now...

File: app/assets/stylesheets/_main.scss [hunk 5]
```diff
@@ -57,24 +57,6 @@ body {
   color: $color-gray-dark !important;
 }
 
-.cf-dropdown {
```
Comment (nolawi) [inline app/assets/stylesheets/_main.scss]: combined in commons

File: client/app/certification/ConfirmHearing.jsx [hunk 6]
```diff
@@ -313,7 +316,7 @@ export class ConfirmHearing extends React.Component {
               also, we should stop using rails pdf viewer */
             <LoadingContainer color={AppConstants.LOGO_COLORS.CERTIFICATION.ACCENT}>
               <iframe
-                className="cf-doc-embed cf-iframe-with-loading form9-viewer"
```
Comment (mdbenjam) [inline client/app/certification/ConfirmHearing.jsx]: is `cf-iframe-with-loading` even defined anywhere?

File: app/views/layouts/certifications.html.erb (hunk 2 above)
Comment (mdbenjam) [reply app/views/layouts/certifications.html.erb]: @amprokop is this used anywhere?

File: app/assets/stylesheets/_commons.scss [hunk 7]
```diff
@@ -105,12 +105,6 @@ p {
   font-weight: 300;
 }
 
-.cf-doc-embed {
```
Comment (mdbenjam) [inline app/assets/stylesheets/_commons.scss]: why don't we need this one anymore?

Comment (mdbenjam) [reply app/assets/stylesheets/_commons.scss]: jk I see you defined iframe.

File: app/assets/stylesheets/reader/_pdf_viewer.scss (hunk 4 above)
Comment (mdbenjam) [reply app/assets/stylesheets/reader/_pdf_viewer.scss]: We shouldn't touch this file. It's not in our code, it's a symbolic link to a library's CSS. PDFJS ships with pretty important CSS for the PDF rendering. I've added a symbolic link from that file to our assets folder so it's included in the asset pipeline, but also gets updated whenever we update the npm module.

File: client/app/certification/ConfirmHearing.jsx [hunk 8]
```diff
@@ -313,7 +316,7 @@ export class ConfirmHearing extends React.Component {
...
+                className="cf-iframe-with-loading" {...form9Viewer}
```
Comment (mdbenjam) [inline client/app/certification/ConfirmHearing.jsx]: the iframe properties are taking precedence over the form9Viewer ones. The pdf has gotten bigger.
screen shot 2018-05-10 at 5 29 42 pm

File: app/assets/stylesheets/react/_rc_collapse.scss [hunk 9]
```diff
@@ -1,36 +1,38 @@
-/*
-* rc-collapse
-* Repo: https://github.com/react-component/collapse
-* This css has been modified to match the accordion for caseflow.
-*/
+//rc-collapse
```
[the remark this answers is not in the document]
Comment (nolawi) [reply app/assets/stylesheets/react/_rc_collapse.scss]: I believe its an overwrite to `https://github.com/react-component/collapse` accordion, essentially its react component styles. I understand it makes sense to keep vendor files separate but in our case it seems overkill to keep vendor folder for file we can't [modify](https://github.com/department-of-veterans-affairs/caseflow/pull/2501/files#r125059132)

File: app/views/layouts/certifications.html.erb (hunk 2 above)
Comment (amprokop) [reply app/views/layouts/certifications.html.erb]: it should not be, we transitioned the whole app to React! thanks for checking

**Comment by nolawi** [comment]
resolves #5163

**Pull request merged**